vite-project/vite.config.js

```diff
@@ -19,40 +19,33 @@ const wasmContentTypePlugin = {
     },
 };
 
-export default defineConfig(({ command }) => {
-    if (command === 'serve') {
-        return {
-            build: {
-                target: 'esnext',
-                rollupOptions: {
-                    external: ['@aztec/bb.js']
-                }
-            },
-            optimizeDeps: {
-                esbuildOptions: {
-                    target: 'esnext'
-                }
+export default defineConfig({
+    build: {
+        target: 'esnext',
+        rollupOptions: {
+            external: ['@aztec/bb.js']
+        }
+    },
+    optimizeDeps: {
+        esbuildOptions: {
+            target: 'esnext'
+        }
+    },
+    plugins: [
+        copy({
+            targets: [{ src: 'node_modules/**/*.wasm', dest: 'node_modules/.vite/dist' }],
+            copySync: true,
+            hook: 'buildStart',
+        }),
+        wasmContentTypePlugin,
+        {
+            name: "configure-response-headers",
+            configureServer: (server) => {
+            server.middlewares.use((_req, res, next) => {
+                res.setHeader("Cross-Origin-Opener-Policy", "same-origin");
+                next();
+            });
             },
-            plugins: [
-                copy({
-                    targets: [{ src: 'node_modules/**/*.wasm', dest: 'node_modules/.vite/dist' }],
-                    copySync: true,
-                    hook: 'buildStart',
-                }),
-                command === 'serve' ? wasmContentTypePlugin : [],
-                {
-                    name: "configure-response-headers",
-                    configureServer: (server) => {
-                    server.middlewares.use((_req, res, next) => {
-                        res.setHeader("Cross-Origin-Embedder-Policy", "require-corp");
```

Comment:
Do you think this is causing the error in firefox? I believe this line helped someone previously enable multithreading in the browser

Reply:
Yeah, I saw that as the reason for it to be there. I've been looking for some other way to fix it on firefox.

Reply:
Apparently, the best way would be to just not have webpack inline the wasm because Firefox imposes tighter security restrictions than chrome. https://chatgpt.com/share/66fc8804-835c-8006-9bac-c2c654a2c7cb

i.e. this suggestion: https://github.com/AztecProtocol/aztec-packages/issues/6527#issuecomment-2384188419